rpi-convert-stage-5.sh

```diff
@@ -57,7 +57,12 @@ build_uboot_files() {
   git checkout $commit >> "$build_log" 2>&1
 
   make --quiet distclean >> "$build_log"
-  make --quiet rpi_3_32b_defconfig >> "$build_log" 2>&1
+  if [ "$2" = "raspberrypi3" ]; then
```

Comment:
Add $2 to the list of arguments the function takes. Add separate variable for 'defconfig' (or name it as you wish) so that you can call `make --quiet $defconfig` only once.

Reply:
Yes, exactly. Only remember about `==` for comparison.